packages/subgraph/src/mapping.ts

```diff
@@ -257,37 +274,37 @@ function executeNonEmptySetBlockNumbersForEpochMessage(
   message.data = reader.diff(snapshot);
 }
 
-function executeEmptySetBlockNumbersForEpochMessage(
-  cache: StoreCache,
-  snapshot: BytesReader,
-  reader: BytesReader,
-  id: String,
-  messageBlock: MessageBlock
-): void {
-  let globalState = cache.getGlobalState();
-  let message = cache.getSetBlockNumbersForEpochMessage(id);
-  message.block = messageBlock.id;
-
-  let numNetworks = BigInt.fromU64(decodeU64(reader));
-  if (!reader.ok) {
-    return;
-  }
-
-  message.count = numNetworks;
-
-  log.warning("BEFORE EPOCH LOOP, AMOUNT TO CREATE: {}", [
-    message.count!.toString()
-  ]);
-
-  for (let i = BIGINT_ZERO; i < message.count!; i += BIGINT_ONE) {
-    log.warning("EPOCH LOOP, CREATING EPOCH: {}", [i.toString()]);
-    let newEpoch = cache.getEpoch(nextEpochId(globalState));
-    globalState.latestValidEpoch = newEpoch.id;
-  }
-  log.warning("AFTER EPOCH LOOP", []);
-
-  message.data = reader.diff(snapshot);
-}
+// function executeEmptySetBlockNumbersForEpochMessage(
+//   cache: StoreCache,
+//   snapshot: BytesReader,
+//   reader: BytesReader,
+//   id: String,
+//   messageBlock: MessageBlock
+// ): void {
+//   let globalState = cache.getGlobalState();
+//   let message = cache.getSetBlockNumbersForEpochMessage(id);
+//   message.block = messageBlock.id;
+//
+//   let emptyEpochCount = BigInt.fromU64(decodeU64(reader));
+//   if (!reader.ok) {
+//     return;
+//   }
+//
+//   message.count = emptyEpochCount;
+//
+//   log.warning("BEFORE EPOCH LOOP, AMOUNT TO CREATE: {}", [
+//     message.count!.toString()
+//   ]);
+//
+//   for (let i = BIGINT_ZERO; i < message.count!; i += BIGINT_ONE) {
+//     log.warning("EPOCH LOOP, CREATING EPOCH: {}", [i.toString()]);
+//     let newEpoch = cache.getEpoch(nextEpochId(globalState, reader));
+//     globalState.latestValidEpoch = newEpoch.id;
+//   }
+//   log.warning("AFTER EPOCH LOOP", []);
+//
+//   message.data = reader.diff(snapshot);
+// }
```

Comment:
Thanks for keeping this around!
If things don't work well with the Subgraph calling the EpochManager, we can revert to the previous behavior.